kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/BatchesResource.scala

```diff
@@ -150,12 +150,11 @@ private[v1] class BatchesResource extends ApiRequestContext with Logging {
     responseCode = "200",
     content = Array(new Content(
       mediaType = MediaType.APPLICATION_JSON)),
-    description = "close a batch session")
+    description = "close and cancel a batch session")
   @DELETE
   @Path("{batchId}")
   def closeBatchSession(
       @PathParam("batchId") batchId: String,
-      @QueryParam("killApp") killApp: Boolean,
       @QueryParam("hive.server2.proxy.user") hs2ProxyUser: String): Response = {
```

Comment:
should we set the return type to `CloseBatchResponse`?

Reply:
so that the response schema is visible in swagger.

Reply:
fixed

Reply:
not sure whether it is needed, you can check the swagger locally


```
      schema = new Schema(implementation = classOf[CloseBatchResponse]))),
```

Reply:
fixed